go-controller/pkg/kubevirt/types.go

```diff
@@ -0,0 +1,43 @@
+package kubevirt
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	OriginalSwitchNameLabel = types.OvnK8sPrefix + "/original-switch-name"
```

Comment:
You can filter by label but not by annotation, and vm pods belonging to a switch is a query that make sense (is not needed right now but maybe in the future), so that's why I added as a label.